protocol/app/prepare/prepare_proposal.go

```diff
@@ -71,8 +74,21 @@ func PrepareProposalHandler(
 			return &EmptyResponse, nil
 		}
 
+		var extCommitBzTx []byte
+		if len(req.Txs) >= slinkyabci.NumInjectedTxs {
```

Comment:
Yeah the prepare proposal handler will inject the VEs as the first tx in the block. If there are no txs, it means there were no VEs injected. This is basically only going to happen if the protocol hasn't enabled VEs yet.

Reply:
can we include this in a comment?

Reply:
Could we also change this constant name? `NumInjectedTxs` doesn't really mean anything. Ideally this would be a function named `TxsContainsVEs` or the constant would be something like `MinTxRequiredForVEEnabled`